src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -451,7 +451,16 @@ private OpenApiOperation CreateBaseOperation(string description, List<OpenApiTag
             if (dbObject.SourceType == EntitySourceType.StoredProcedure)
             {
                 Entity entityTest = _runtimeConfig.Entities[entityName];
-                List<SupportedHttpVerb> spRestMethods = entityTest.Rest.Methods.ToList();
+
+                List<SupportedHttpVerb>? spRestMethods;
+                if (entityTest.Rest.Methods is not null)
```

Comment:
nit: i don't think i had a reason to continue using the name `entityTest`. You can just revise it to be `entity`.